src/app/services/component-communication.service.ts

```diff
@@ -31,11 +35,16 @@ export class ComponentCommunicationService {
 }
 
 export class CpeEvent {
-    constructor(public name: Events, public value?: any) { }
+    constructor(public name: Events, public status: Status, public value?: any) { }
 }
 
 // possible events that can be emitted.
 export enum Events {
-    searchExecuted
+    searchExecuted,
+    changeBackground
 }
 
+export enum Status {
+    starting,
```

Comment:
maybe we should stick to past tense, so in this case "started"

Reply:
That sounds better -> https://github.com/dasch-swiss/dsp-cpe/pull/11/commits/ce69a805ba4a8febc3b83d1ebc2d204ee1071542